### PR TITLE
Adding node-postgres driver examples

### PR DESCRIPTION
**Pull request by HarshDaryani896**
Added node-postgres driver examples to be run in the jenkins pipeline.

### CODE REVIEW

File: node-postgres/run-app.sh [hunk 1]
```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -e
+
+CURRENT_DIR=`dirname $0`
+ARTIFACTS_PATH=`realpath $CURRENT_DIR`
+YUGABYTE_HOME_DIRECTORY=/home/ec2-user/code/yugabyte-binary/yugabyte-2.19.0.0
```
Comment (ashetkar) [inline node-postgres/run-app.sh]: Why do you need yugabyte version 2.19.0.0 hard-coded? The pipeline will take the one which is configured for the build.

Comment (HarshDaryani896) [reply node-postgres/run-app.sh]: This was required for local testing, have removed them.

File: node-postgres/run-app.sh [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+npm install @yugabytedb/pg-pool
+
+echo "Cloning the driver examples repository"
+
+git clone git@github.com:yugabyte/driver-examples.git && cd driver-examples/nodejs
+npm install
+
+echo "Exporting environment variable YB_PATH with the value of the path of the YugabyteDB installation directory."
+
+export YB_PATH="$YUGABYTE_HOME_DIRECTORY"
```
Comment (ashetkar) [inline node-postgres/run-app.sh:19]: `YB_PATH ` - Where is this being used?

Comment (HarshDaryani896) [reply node-postgres/run-app.sh:19]: In the test code, this is used to start the cluster.

**Review by ashetkar — APPROVED** [review]

**Pull request merged**